tests/test_fields.py

```diff
@@ -498,7 +498,7 @@ class TestIntegerRangeField(FieldValues):
         (NumericRange(**{'lower': '1', 'upper': '2'}),
          {'lower': 1, 'upper': 2, 'bounds': '[)'}),
         (NumericRange(**{'empty': True}), {'empty': True}),
-        (NumericRange(), {'bounds': '[)', 'lower': None, 'upper': None}),
+        (NumericRange(), {'bounds': '()', 'lower': None, 'upper': None}),
```

Comment:
Since I suggested to not change our range fields, this tests should cover both `psycopg 2 and 3` with another way e.g. `(NumericRange(bounds='()'), {'bounds': '()', 'lower': None, 'upper': None}),`

I know that if there are users that use `drf-extra-fields` and has this exact same test in their project and if they switch to `Django 4.2 + psycopg 3`, their test will fail. But I think this is their problem since it's caused by a change about how `psycopg 3` handles bounds. Our serializer fields just return what it's given to them.

Also, postgresql already converts bounds on infinity values to `(` and `)` (like `psycopg 3`) as far as I understand from [this page](https://www.postgresql.org/docs/11/rangetypes.html). So it won't be a problem for users that use these fields to represent data from their database I guess. Quote from that page:
> Specifying a missing bound as inclusive is automatically converted to exclusive, e.g., [,] is converted to (,).